test/regressions/webpack.config.js

```diff
@@ -32,11 +33,12 @@ module.exports = {
     rules: [
       {
         test: /\.(js|ts|tsx)$/,
-        exclude: /node_modules/,
+        exclude: /node_modules|prism\.js/,
```

Comment:
Are we excluding this so that it picks the `prism.mjs` (ES Module) file?

Reply:
It's because [`@mui/markdown/prism`](https://github.com/mui/material-ui/blob/master/packages/markdown/prism.js) gets imported in several contexts ESM and Commonjs. It's a commonjs file but webpack resolves it as ESM and fails to find exports. Ideally we move this to ESM as well, but I don't want to expand the scope of this PR further

Reply:
I'm adding a comment